src/vm/loaderallocator.cpp

```diff
@@ -1877,6 +1933,32 @@ void AssemblyLoaderAllocator::ReleaseManagedAssemblyLoadContext()
     }
 }
 
+#ifdef FEATURE_COMINTEROP
+// TODO: should it rather go to AssemblyLoaderAllocator?
+ComCallWrapperCache * LoaderAllocator::GetComCallWrapperCache()
+{
+    CONTRACTL
+    {
+        THROWS;
+        GC_TRIGGERS;
+        MODE_ANY;
+        INJECT_FAULT(COMPlusThrowOM(););
+    }
+    CONTRACTL_END;
+
+    if (!m_pComCallWrapperCache)
```

Comment:
Can we fix up the alignment below.

Reply:
Oops, this needs a lock. I've forgotten about this TODO of mine. I am glad you'd noticed the misalignment that made me look at this.